### PR TITLE
Update BCD info, part 39

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Adding to #19185 

The PR updates BCD info in WebAPI docs.

Simple headers are done. Now files with inline badges left.

### CODE REVIEW

File: files/en-us/web/api/elementinternals/role/index.md [hunk 1]
```diff
@@ -8,9 +8,10 @@ tags:
   - Reference
   - role
   - ElementInternals
+  - Experimental
 browser-compat: api.ElementInternals.role
 ---
-{{DefaultAPISidebar("DOM")}}
+{{APIRef("DOM")}}{{SeeCompatTable}}
```
Comment (jpmedley) [inline files/en-us/web/api/elementinternals/role/index.md:14]:
```suggestion
{{APIRef("DOM")}}
```

Comment (jpmedley) [reply files/en-us/web/api/elementinternals/role/index.md:14]: We have been removing this macro for some time now. Please change this throughout.

File: files/en-us/web/api/gamepadpose/index.md [hunk 2]
```diff
@@ -14,29 +14,29 @@ tags:
   - WebVR
 browser-compat: api.GamepadPose
 ---
-{{securecontext_header}}{{APIRef("Gamepad API")}}{{SeeCompatTable}}
+{{SecureContext_Header}}{{APIRef("Gamepad API")}}{{SeeCompatTable}}
```
Comment (jpmedley) [inline files/en-us/web/api/gamepadpose/index.md:17]:
```suggestion
{{SecureContext_Header}}{{APIRef("Gamepad API")}}
```

File: files/en-us/web/api/hid/index.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ tags:
   - Experimental
 browser-compat: api.HID
 ---
-{{securecontext_header}}{{APIRef("WebHID API")}}{{SeeCompatTable}}
+{{SecureContext_Header}}{{APIRef("WebHID API")}}{{SeeCompatTable}}
```
Comment (jpmedley) [inline files/en-us/web/api/hid/index.md:13]:
```suggestion
{{SecureContext_Header}}{{APIRef("WebHID API")}}
```

Comment (jpmedley) [reply files/en-us/web/api/hid/index.md:13]: Please remove any others that are in the docs you touched.

**Review by jpmedley — CHANGES_REQUESTED** [review]
Thank you for picking up some of our busy work. This looks pretty good, but I have one nit. We are gradually trying to remove the `{{SeeCompatTable}}` macros. 

FYI, our macro engine is case insensitive so there's no need to correct macro names.

File: files/en-us/web/api/elementinternals/role/index.md (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/api/elementinternals/role/index.md:14]: Why are we removing them? These are experimental features, isn't it?

I've not heard about this.

Comment (jpmedley) [reply files/en-us/web/api/elementinternals/role/index.md:14]: Some experimental features are added to existing interfaces. This makes it hard to use in a consistently meaningful way. The inline macros you have been so kind as to add are better.

**Comment by OnkarRuikar** [comment]
I've removed all the `{{SeeCompatTable}}` macros from this PR.

@jpmedley
> We are gradually trying to remove the `{{SeeCompatTable}}` macros.

1. Do we want to remove `{{Deprecated_Header}}` and `{{Non-standard_Header}}` macros as well?
2. Does this apply to the entire MDN Web Docs or just Web API pages? For example an experimental CSS page: https://developer.mozilla.org/en-US/docs/Web/CSS/masonry-auto-flow
3. Why gradually? Are we following any criteria/procedure to remove the `{{SeeCompatTable}}` macro? I mean we can simply remove them.

> Please remove any others that are in the docs you touched.

May I remove not just mine but all the `{{SeeCompatTable}}` macros. It will be easier that way and can be done with less effort as the script is ready.

**Comment by teoli2003** [comment]
We are not removing the `{{SeeCompatMacro}}`.

We are synchronizing info in BCD and MDN.

**Comment by OnkarRuikar** [comment]
I am confused now. 😵‍💫

What I gathered from Joe's comments is that we are not adding new `{{SeeCompatTable}}` macro calls anymore and removing them when a feature is no longer experimental. Thus one day there will be no `SeeCompatTable` calls in the entire repo. Correct me.

To avoid confusions do we want to rename it to: `{{Experimental_Header}}`?

Let me know I what changes are required from me to land this PR.

**Comment by teoli2003** [comment]
> I am confused now. 😵‍💫
> 
> What I gathered from Joe's comments is that we are not adding new `{{SeeCompatTable}}` macro calls anymore and removing them when a feature is no longer experimental. Thus one day there will be no `SeeCompatTable` calls in the entire repo. Correct me.
> 
> To avoid confusions do we want to rename it to: `{{Experimental_Header}}`?
> 
> Let me know I what changes are required from me to land this PR.

We are adding `{{SeeCompatData}}` to every experimental feature. As far as I know, there has been no discussion about removing the macro. (I've edited the previous post to be more explicit)

File: files/en-us/web/api/featurepolicy/index.md [hunk 4]
```diff
@@ -13,6 +13,7 @@ tags:
   - Reference
   - access
   - delegation
+  - Experimental
 browser-compat: api.FeaturePolicy
 ---
 {{APIRef("Feature Policy")}}
```
Comment (teoli2003) [inline files/en-us/web/api/featurepolicy/index.md:19]:
```suggestion
{{APIRef("Feature Policy")}}{{SeeCompatMacro}}
```

File: files/en-us/web/api/filesystemwritablefilestream/index.md [hunk 5]
```diff
@@ -9,9 +9,10 @@ tags:
   - Interface
   - stream
   - write file
+  - Experimental
 browser-compat: api.FileSystemWritableFileStream
 ---
-{{securecontext_header}}{{DefaultAPISidebar("File System Access API")}}
+{{SecureContext_Header}}{{APIRef("File System Access API")}}
```
Comment (teoli2003) [inline files/en-us/web/api/filesystemwritablefilestream/index.md:15]:
```suggestion
{{SecureContext_Header}}{{APIRef("File System Access API")}}{{SeeCompatData}}
```

File: files/en-us/web/api/encodedvideochunk/index.md [hunk 6]
```diff
@@ -7,31 +7,32 @@ tags:
   - Interface
   - Reference
   - EncodedVideoChunk
+  - Experimental
 browser-compat: api.EncodedVideoChunk
 ---
-{{DefaultAPISidebar("WebCodecs API")}}
+{{APIRef("WebCodecs API")}}
```
Comment (teoli2003) [inline files/en-us/web/api/encodedvideochunk/index.md:13]:
```suggestion
{{APIRef("WebCodecs API")}}{{SeeCompatData}}
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Here is what is needed to have this PR synchronized with BCD data.

**Comment by teoli2003** [comment]
We are waiting for an answer from @jpmedley here.

We are only mirroring the flags that are on mdn/browser-compat-data, so that they are the same and we can, in the future maintain their synchronization.

**Comment by teoli2003** [comment]
So this is the last PR so that mdn/browser-compat-data and mdn/content are synchronized (wrt to these badges/banners). 

Now you only need to run your sync script once a week, usually on Wednesdays?

Am I right, @OnkarRuikar

**Comment by OnkarRuikar** [comment]
> Now you only need to run your sync script once a week, usually on Wednesdays?

How frequently we want this to run?

Currently I've set it up to run within 24 hrs of a new BCD release. On every boot my machine looks for new version:
```
> curl https://raw.githubusercontent.com/mdn/browser-compat-data/main/package.json 2>&1 | grep -Po '(?<=version....)[\d\.]+'
5.2.0
```

With this our content will be more up-to-date than other dedicated browser compatibility websites(e.g. can i use).
We can also run it daily, without waiting for a release, like the markdownlint cron job, to be the most up-to-date...

**Comment by teoli2003** [comment]
Once a week, on Wednesdays: usually, the bcd package is produced on Tuesday and used by Yari on Wednesday afternoon. So if we update mdn/content in between, there are 0 discrepancies.

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by teoli2003** [comment]
ping @jpmedley

**Pull request closed without merging**